python/docs/source/user_guide/ansi_migration_guide.ipynb

```diff
@@ -0,0 +1,193 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4fa81d13",
+   "metadata": {},
+   "source": [
+    "# ANSI Migration Guide - Pandas API on Spark\n",
+    "ANSI mode is now on by default for Pandas API on Spark. This guide helps you understand the key behavior differences you’ll see.\n",
+    "In short, with ANSI mode on, Pandas API on Spark behavior matches native pandas in cases where Pandas API on Spark with ANSI off did not."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e1c7952",
+   "metadata": {},
+   "source": [
+    "## Behavior Change\n",
+    "### String Number Comparison\n",
+    "**ANSI off:** Spark implicitly casts numbers and strings, so `1` and `'1'` are considered equal.\n",
+    "\n",
+    "**ANSI on:** behaves like pandas, `1 == '1'` is False."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "69474e28-c1cd-40fe-8ec6-7373b56c4dee",
+   "metadata": {},
+   "source": [
+    "Examples are as shown below:\n",
+    "\n",
+    "```python\n",
+    ">>> pdf = pd.DataFrame({\"int\": [1, 2], \"str\": [\"1\", \"2\"]})\n",
+    ">>> psdf = ps.from_pandas(pdf)\n",
+    "\n",
+    "# ANSI on\n",
+    ">>> psdf[\"int\"] == psdf[\"str\"]\n",
```

Comment:
Shall we add the configuration here to make it clear?

```suggestion
    "# ANSI on\n",
    ">>> spark.conf.set(\"spark.sql.ansi.enabled\", True)\n",
    ">>> psdf[\"int\"] == psdf[\"str\"]\n",
```

same for the similar places below.